mawabot/cogs/info.py

```diff
@@ -161,6 +183,43 @@ def _get_members(self, guild, names):
         uptime = str(self.bot.uptime).split('.')[0]
         await ctx.message.edit(content=f'`{uptime}`')
 
+    @commands.command()
+    async def stats(self, ctx):
+        ''' Gets bot stats '''
+
+        title = 'mawabot'
+        url = 'https://github.com/strinking/mawabot'
+
+        uptime = str(self.bot.uptime).split('.')[0]
+        channels = sum(1 for _ in self.bot.get_all_channels())
+        desc = [f'Uptime: `{uptime}`',
+                f'Guilds: `{len(self.bot.guilds)}`',
+                f'Channels: `{channels}`',
+                f'Users: `{len(self.bot.users)}`',]
+
+        embed = discord.Embed(title=title, url=url, description='\n'.join(desc))
+        git = []
+        contributors = await self.get_git_contributors()
+
+        if contributors:
+            contributors = sorted(contributors, key=lambda x: x.get('total'), reverse=True)
+            for user in contributors:
+                author = user['author']
+                git.append(f'[{author["login"]}]({author["html_url"]}) ({user["total"]})')
+
+        embed.add_field(name='Repo Info', value=f'Version: `{version}`\n{", ".join(git)}')
+
+        # Get system info
+        pid = os.getpid()
+        py = psutil.Process(pid)
+
+        cpu = py.cpu_percent()
+        mem = py.memory_info()[0]/1000/1000
```

Comment:
I think `py.memory_info()[0] / MB` would be more clear.

Also imo we should use MiB not MB.
```python
MB = 1000 *1000
MiB = 1024 * 1024
```